llvm/docs/RFCProcess.rst

```diff
@@ -0,0 +1,80 @@
+=================================
+Request For Comment (RFC) process
+=================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Introduction
+============
+Substantive changes to LLVM projects need to be acceptable to the wider
+community, which requires gaining community consensus to adopt the changes.
+This is done by posting an RFC and obtaining feedback about the proposal.
+
+Process
+=======
+
+Writing an RFC
+--------------
+The process begins with writing a proposal for the changes you'd like to see
+made. The proposal should include:
+
+* a high-level overview of what changes are being proposed,
+* information detailing the motivation for why the changes are being proposed,
```

Comment:
Maybe skip some extra words - this bullet point could be "the motivation for why the changes are being proposed"? and the previous might be "an overview of the proposed changes"

Reply:
Done, though I went with a "detailed overview" to imply there needs to be more than just "I had a thought" level of detail involved.